xbmc/utils/log.h

```diff
@@ -113,9 +113,9 @@ class CLog : public ISettingsHandler, public ISettingCallback
   }
 
 #define LogF(level, format, ...) \
-  Log((level), ("{}: " DEF_TO_STR_VALUE(format)), __FUNCTION__, ##__VA_ARGS__)
+  Log((level), ("{}: " format), __FUNCTION__, ##__VA_ARGS__)
 #define LogFC(level, component, format, ...) \
-  Log((level), (component), ("{}: " DEF_TO_STR_VALUE(format)), __FUNCTION__, ##__VA_ARGS__)
+  Log((level), (component), ("{}: " format), __FUNCTION__, ##__VA_ARGS__)
```

Comment:
I'm not sure if `DEF_TO_STR_VALUE` macro can be removed because not even sure why is here....

This ends up adding quotes that weren't there before:

`DEBUG <general>: CWIN32Util::GetWindowsHDRStatus: "Display is HDR capable and current HDR status is OFF"`

`DEBUG <general>: DXVA::CVideoBufferPool::~CVideoBufferPool: "destructing buffer pool."`

Reply:
@lrusak 
This is intentional or expected behavior?  It was tested? (runtime)